src/overlays/actors/ovl_Bg_Ikninside/z_bg_ikninside.h

```diff
@@ -5,14 +5,16 @@
 
 struct BgIkninside;
 
+#define DMIKNINSIDE_GET_SWITCH(thisx) (((thisx)->params & 0xFE00) >> 9)
+
 typedef void (*BgIkninsideActionFunc)(struct BgIkninside*, GlobalContext*);
 
 typedef struct BgIkninside {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x68];
-    /* 0x01AC */ BgIkninsideActionFunc actionFunc;
+    /* 0x000 */ DynaPolyActor dyna;
+    /* 0x15C */ ColliderCylinder collider;
+    /* 0x1A8 */ UNK_TYPE1 pad_1A8[2];
+    /* 0x1AA */ s16 unk_1AA;
```

Comment:
Looks like this is a timer, so can name it.
```suggestion
    /* 0x1AA */ s16 timer;
```